core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -358,14 +373,15 @@ class TechManager {
         researchedTechnologies.addAll(techsResearched.map { getRuleset().technologies[it]!! })
         researchedTechUniques.addAll(researchedTechnologies.asSequence().flatMap { it.uniqueObjects.asSequence() })
         updateTransientBooleans()
+        movementSpeedOnRoads = if (civInfo.hasUnique("Improves movement speed on roads"))
```

Comment:
This should be in updateTransientBooleans, not in setTransients

Reply:
... where I had it initially until I thought "wait a minute, it's no longer a Boolean, so..." I think I checked they're 1:1 at the moment anyway. Will look more closely later.

Reply:
No I was wrong, my placement would have omitted an update on addTechnology() where it _is_ needed.